plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -502,41 +503,41 @@ public void testMySqlDatetimeType(ZoneId sessionZone)
                 // time double in Vilnius
                 .addRoundTrip("datetime(3)", "TIMESTAMP '2018-10-28 03:33:33.333'", TIMESTAMP_MILLIS, "TIMESTAMP '2018-10-28 03:33:33.333'")
                 // epoch
-                .addRoundTrip("datetime(3)", "TIMESTAMP '1970-01-01 00:00:00.000'", TIMESTAMP_MILLIS, "TIMESTAMP '1970-01-01 01:00:00.000'")
+                .addRoundTrip("datetime(3)", "TIMESTAMP '1970-01-01 01:00:00.000'", TIMESTAMP_MILLIS, "TIMESTAMP '1970-01-01 01:00:00.000'")
                 // TODO time gaps do not round-trip (https://github.com/trinodb/trino/issues/6910)
-                .addRoundTrip("datetime(3)", "TIMESTAMP '1970-01-01 00:13:42.000'", TIMESTAMP_MILLIS, "TIMESTAMP '1970-01-01 01:13:42.000'")
-                .addRoundTrip("datetime(3)", "TIMESTAMP '2018-04-01 02:13:55.123'", TIMESTAMP_MILLIS, "TIMESTAMP '2018-04-01 03:13:55.123'")
+                .addRoundTrip("datetime(3)", "TIMESTAMP '1970-01-01 01:13:42.000'", TIMESTAMP_MILLIS, "TIMESTAMP '1970-01-01 01:13:42.000'")
```

Comment:
Don't "fix" these. Take a look at the TODO here. `00:13:42.000` should be read back as `00:13:42.000` instead of the 1 hour DST jump.